src/components/ThirdPartyCollectionDetailPage/CollectionContextMenu/CollectionContextMenu.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { Dropdown, Button, Icon } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { getCollectionEditorURL, getExplorerURL, isLocked } from 'modules/collection/utils'
+import ConfirmDelete from 'components/ConfirmDelete'
+import { Props } from './CollectionContextMenu.types'
+import './CollectionContextMenu.css'
+
+export default class CollectionContextMenu extends React.PureComponent<Props> {
+  handleNavigateToExplorer = () => {
+    const { collection } = this.props
+    this.navigateTo(getExplorerURL(collection), '_blank')
+  }
+
+  handleNavigateToEditor = () => {
+    const { collection, items, onNavigate } = this.props
+    onNavigate(getCollectionEditorURL(collection, items))
+  }
+
+  handleDeleteCollection = () => {
+    const { collection, onDelete } = this.props
+    onDelete(collection)
+  }
+
+  handleEditURN = () => {
+    // this.props.onOpenModal('EditURNModal')
+  }
+
+  navigateTo = (url: string, target: string = '') => {
+    const newWindow = window.open(url, target)
+    if (newWindow) {
+      newWindow.focus()
+    }
+  }
+
+  render() {
+    const { collection } = this.props
+    return (
+      <Dropdown
+        className="CollectionContextMenu"
+        trigger={
+          <Button basic>
+            <Icon name="ellipsis horizontal" />
+          </Button>
+        }
+        inline
+        direction="left"
+      >
+        <Dropdown.Menu>
+          <Dropdown.Item text={t('collection_context_menu.see_in_world')} onClick={this.handleNavigateToExplorer} />
+          <Dropdown.Item text={t('collection_context_menu.open_in_editor')} onClick={this.handleNavigateToEditor} />
+
+          {!isLocked(collection) ? (
```

Comment:
Can you add a TODO here to check for the items to see if any of them were already published? The idea is to disable the deletion of the collection if this happens.